### ISSUE TITLE
Rusher not following Rushee

### ISSUE THREAD

**Issue by SteelCityVII** [kolton/d2bot-with-kolbot#2695]
I checked all the scripts several times, the rusher doesn't open the game at all and the rushee just sits in town.

**Comment by gtoilet** [comment]
leader blank ? party settings correct ? rushee is supposed to make the game not rusher

**Comment by SteelCityVII** [comment]
Yeah all correct, so confusing as to why it's not working. Rushee makes the game as well.
